From ab744f985818ae323904135721dd1e3ff9893710 Mon Sep 17 00:00:00 2001
From: Ranjan Dutta <ranjan.dutta@intel.com>
Date: Sat, 14 May 2022 22:57:49 +0800
Subject: [PATCH 44/58] Revert "drm/amd/pm: return -ENOTSUPP if there is no
 get_dpm_ultimate_freq function"

This reverts commit 6e669baa3363406c4f600bce95756e58123b19c4.
---
 drivers/gpu/drm/amd/pm/swsmu/amdgpu_smu.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/amd/pm/swsmu/amdgpu_smu.c b/drivers/gpu/drm/amd/pm/swsmu/amdgpu_smu.c
index ee27970cfff95..e5893218fa4bb 100644
--- a/drivers/gpu/drm/amd/pm/swsmu/amdgpu_smu.c
+++ b/drivers/gpu/drm/amd/pm/swsmu/amdgpu_smu.c
@@ -115,7 +115,7 @@ int smu_get_dpm_freq_range(struct smu_context *smu,
 			   uint32_t *min,
 			   uint32_t *max)
 {
-	int ret = -ENOTSUPP;
+	int ret = 0;
 
 	if (!min && !max)
 		return -EINVAL;
-- 
2.27.0

